docs/version.py

```diff
@@ -11,7 +11,7 @@
 
 def _get_releases() -> List[Version]:
     """Return reversed sorted list of release tag names."""
-    tags = git.Repo('..').tags
+    tags = git.Repo(search_parent_directories=True).tags
     versions = []
     for t in tags:
         try:
```

Comment:
Cannot comment below but I think the `minor_releases` method is unused. We can also remove it.

Reply:
Why do you think it is unused?

Reply:
sorry, my mistake, I see now it is used in the `conf.py`.